buycex/versions.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "../versions.schema.json",
+  "chain_name": "buycex",
+  "versions": [
+    {
+      "name": "v1.0.0",
+      "recommended_version": true,
```

Comment:
recommended_version refers to the specific version (can be a minor or patch version) among a set of versions that are all compatible with v1.0.0. so "recommended_version" should be something like `"v1.0.0"`, and "compatible_versions": would be
```
[
  "v1.0.0"
]   
```